docs/builtins.md

```diff
@@ -0,0 +1,12 @@
+## Parsing URLs
+
+```
+const url = new URL("https://foo.com/path?foo=bar");
+url.hostname; // foo.com
+url.pathname; // /path
+url.searchParams.get("foo"); //bar
+```
+
+## Internationalization
+
+`Intl` can do a lot nowadays
```

Comment:
Was this file intentional? The URL bit looks pretty useful but guessing the intl bit is WIP!

Reply:
Indeed it is :)